src/app/components/system/README.md

```diff
@@ -0,0 +1,4 @@
+// README.md
+// to-do: create design system
+
+This directory stores all Design System related things
```

Comment:
let's make it a package. what is the process like on public npm? do we need to apply for that ability first?